cmd/caa-log-checker/main.go

```diff
@@ -62,12 +63,12 @@ func checkIssuances(scanner *bufio.Scanner, checkedMap map[string][]time.Time, t
 			continue
 		}
 		if len(matches) != 2 {
-			return fmt.Errorf("line %d: unexpected number of regex matches", lNum)
+			return false, fmt.Errorf("line %d: unexpected number of regex matches", lNum)
```

Comment:
These should `return foundErrors, fmt.Errorf("...")`, since it may have found an error prior to finding an unparseable line.

Reply:
I'm happy to make these return `foundErrors`, but in general all of these cases would result in a non-zero exit code anyway since cmd.FailOnError would be called and that calls `os.Exit(1)`.

Reply:
Yeah, I'd still do it for the sake of defense-in-depth, or making sure the right thing happens if the error semantics of this function are ever changed (e.g. to make it more resilient to single-line parse errors and try to keep going).